src/compiler/types.ts

```diff
@@ -8882,13 +8793,6 @@ export interface NodeFactory {
 
     /** @internal */ createRedirectedSourceFile(redirectInfo: RedirectInfo): SourceFile;
 
-    /** @deprecated @internal */ createUnparsedSource(prologues: readonly UnparsedPrologue[], syntheticReferences: readonly UnparsedSyntheticReference[] | undefined, texts: readonly UnparsedSourceText[]): UnparsedSource;

```

Comment:
So happy to see this stuff disappear 😄

Reply:
This and bunch of bundle manipulation to make sure emit helpers and stuff like that was ordered correctly.